runtime/themes/onedark.toml

```diff
@@ -36,6 +36,16 @@
 "markup.link.url" = { fg = "cyan", modifiers = ["underlined"]}
 "markup.link.label" = { fg = "purple" }
 
+# TODO
+"markup.heading" = "blue"
```

Comment:
onedark already has these styles right above so this can be removed.

It also uses `markup.link.label` instead of `markup.link.text`

Reply:
Fixed!

https://github.com/helix-editor/helix/blob/f85b0ad2695ab91341b65138d2bbb83560e28490/runtime/themes/onedark.toml#L37